api/account.ts

```diff
@@ -46,36 +46,51 @@ class LoginMessage {
     }
 }
 
-export function doCreateAccount(alias: string, ethAddress: string, message: string, hexSignature: string): any {
+export function doCreateAccount(alias: string, ethAddress: string, message: any, hexSignature: string): any {
     // check signature
-    let messageBytes = ethers.utils.arrayify(message);
+    const message_ = JSON.parse(JSON.stringify(message));
```

Comment:
The message should be a  json string, so the no need to stringfy it, just parse it directly.  the message's type should be string.